packages/block-renderer/src/components/patterns-renderer-provider.tsx

```diff
@@ -19,7 +18,7 @@ const PatternsRendererProvider = ( {
 	children,
 	siteInfo = {},
 	shouldShufflePosts,
-}: Props ) => {
+}: PropsWithChildren< Props > ) => {
```

Comment:
The `PropsWithChildren` change isn't strictly necessary anymore, but it's a more correct definition, so I think we should leave it in.